config/skim_UL17_uhh.cfg

```diff
@@ -2,11 +2,11 @@
 # do not select events with at least two jets in the final state
 # false : only >= 2 b jet ; true : all events
 ## for sync use beInclusive==true, for full porduction use false
-beInclusive = true
+beInclusive = false
```

Comment:
Please set this flag to true: we need to keep the events without 2 AK4 b-jets as they may contain an AK8 b-jet.

Reply:
Set back to true :+1: